android-appcoins-billing/src/main/java/com/appcoins/sdk/billing/helpers/AppCoinsPendingIntentCaller.java

```diff
@@ -0,0 +1,51 @@
+package com.appcoins.sdk.billing.helpers;
+
+import android.app.Activity;
+import android.app.PendingIntent;
+import android.content.Intent;
+import android.content.IntentSender;
+import android.os.Bundle;
+import android.os.Parcelable;
+import java.util.HashMap;
+import java.util.Map;
+
+public class AppCoinsPendingIntentCaller {
+
+  public static final String BUY_INTENT_RAW = "BUY_INTENT_RAW";
+  public static final String BUY_INTENT = "BUY_INTENT";
+  public static final AppCoinsPendingIntentCaller instance =
```

Comment:
Instead of having a public variable maybe use a getInstance() method for better code clear